01 - JavaScript Drum Kit/index-START.html

```diff
@@ -59,6 +59,26 @@
 
 <script>
 
+    function playSound(e){
+        const audio = document.querySelector(`audio[data-key="${e.keyCode}"]`);
+        const key = document.querySelector(`.key[data-key="${e.keyCode}"]`);
+        if (!audio) return; //if audio doesnt exist, end function
```

Comment:
You'd save time if you return for no audio before loading the `key` const. If there's no audio, you're not using the key anyway.